app/src/main/java/com/blackboxteam/app/blackbox/LaunchScreenActivity.java

```diff
@@ -0,0 +1,31 @@
+package com.blackboxteam.app.blackbox;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+
+public class LaunchScreenActivity extends AppCompatActivity implements View.OnClickListener {
+
+    Button nextScreen;
```

Comment:
Try to always set the scope explicitly by specifying the access modifier `private`, `protected` or `public`. :)